docs/python/environments.md

```diff
@@ -16,20 +16,22 @@ To select a specific interpreter, select the **Python: Select Interpreter** comm
 
 ![Python: Select Interpreter command](images/environments/select-interpreters-command.png)
 
-This command automatically looks for and displays a list of available Python interpreters.
+This command automatically looks for and displays a list of available Python interpreters, as well as Conda and virtual environments:
 
 ![List of interpreters](images/environments/interpreters-list.png)
 
-Selecting an interpreter from the list configures your User [Settings](/docs/getstarted/settings.md) accordingly. The Status Bar shows the current interpreter and when selected brings up a list of available interpreters. The Status Bar also reflects when no interpreter is selected.
-
-![No interpreter selected](images/environments/no-interpreter-selected-statusbar.png)
-
 The extension looks for interpreters in the following locations:
 
 - Standard paths such as `/usr/local/bin`, `/usr/sbin`, `/sbin`, `c:\\python27`, `c:\\python36`, etc.
-- Virtual environments located under the workspace (project) folder
+- Virtual environments located under the workspace (project) folder.
 - Conda environments
 
+Selecting an interpreter from the list configures your User [Settings](/docs/getstarted/settings.md) accordingly. The Status Bar shows the current interpreter and when selected brings up a list of available interpreters. The Status Bar also reflects when no interpreter is selected.
+
+![No interpreter selected](images/environments/no-interpreter-selected-statusbar.png)
+
+The currently-selected interpreter is applied when right-clicking a file and selecting **Run Python File in Terminal**. You can also use **Python: Create Terminal** to create a terminal with the current environment activated. Note that setting `python.terminal.activateEnvironments` to `false` skips activating virtual and conda environments before running these commands.
```

Comment:
The currently selected interpreter...
I don't think you need the hyphen.
Also should "condo environments" be "Condo environments"?

Reply:
Not according to the official docs: https://conda.io/docs/user-guide/tasks/manage-environments.html#building-identical-conda-environments